docs/csharp/language-reference/keywords/interpolated-strings.md

```diff
@@ -47,6 +47,16 @@ If the interpolated string contains other characters with special meaning in an
 
 [!code-cs[interpolated-strings4](../../../../samples/snippets/csharp/language-reference/keywords/interpolated-strings4.cs#1)]  
 
+Verbatim interpolated strings use the `@` character in addition to using the `$` character for interpolated strings. You can learn more about verbatim strings in the [string](strings.md) topic. The following code is a modified version of the previous snippet using a verbatim interpolated string:
```

Comment:
For the first sentence, it seems sufficient to say, "Verbatim interpolated strings use the `@` character followed by the `$` character."

Reply:
Maybe introduce the concept first? Like, "You can also create verbatim interpolated strings" or something like that.

Also, for the second sentence, the boilerplate sentence is "For more information about x, see y."